### DISCUSSION TITLE
Reorganise files in originals as though they were imported

### DISCUSSION THREAD

**Discussion by mpfl** [photoprism/photoprism#2208] [Q&A]
I'm pulling photos from NextCloud via the WebDAV sync functionality in PhotoPrism.

Unfortunately, PhotoPrism is pulling them into a folder structure that mirrors NextCloud. I would rather the photos be reorganised as though they had been brought in using the Import function.

Is there anyway to trigger PhotoPrism to reorganise a given subfolder to match the default folder/file naming convention?

**Comment by graciousgrey** [comment]
You could sync the files from Nextcloud to PhotoPrism's import directory instead of originals if you want PhotoPrism to organize them by year/month (see https://docs.photoprism.app/user-guide/library/)
